7.2.0/h2/identity_db_cleanup.sh

```
@@ -0,0 +1,129 @@
+    for table in "${DELETE_ORDER[@]}"; do
+      if [[ "$table" == "IDN_ORG_USER_INVITATION" ]]; then
+        DELETE_PROCEDURE+="DELETE FROM $table WHERE INVITED_ORG_ID = '$ORG_UUID';"
+      elif [[ "$table" == "IDN_OAUTH_PAR" ]]; then
+        DELETE_PROCEDURE+="DELETE FROM $table WHERE CLIENT_ID IN (SELECT CONSUMER_KEY FROM IDN_OAUTH_CONSUMER_APPS WHERE TENANT_ID = $TENANT_ID);"
+      else
+        DELETE_PROCEDURE+="DELETE FROM $table WHERE TENANT_ID = $TENANT_ID;"
+      fi
+    done
+    DELETE_PROCEDURE+="COMMIT; SET REFERENTIAL_INTEGRITY TRUE;"
+
+    echo "$DELETE_PROCEDURE" | java -cp "$H2_JAR_PATH" org.h2.tools.Shell -url "$DB_URL" -user "$DB_USER" -password "$DB_PASSWORD" 2>&1 > /tmp/sql_output.log
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)